packages/veritone-react-common/src/components/FilePicker/FileList/styles.scss

```diff
@@ -15,6 +15,8 @@ $preview-size: 40px;
   position: relative;
   overflow-y: scroll;
   margin-left: 15px;
+  overflow: auto;
```

Comment:
just update the overflow-y above to `auto` instead of overriding it here

Reply:
updated.